.travis.yml

```diff
@@ -20,50 +20,92 @@ script: 'SPEC_OPTS="--format documentation" bundle exec rake validate lint spec
 matrix:
   fast_finish: true
   include:
-  - rvm: 2.1.9
-    env: PUPPET_GEM_VERSION="~> 4"
   - rvm: 2.4.4
     env: PUPPET_GEM_VERSION="~> 5"
-  - rvm: 2.4.2
+  - rvm: 2.5.1
+    env: PUPPET_GEM_VERSION="~> 6" FIXTURES_YML=".fixtures-puppet6.yml"
+  - rvm: 2.4.4
+    sudo: required
+    services: docker
+    env: BEAKER_set="centos-6" BEAKER_PUPPET_COLLECTION=puppet5
+    bundler_args:
+    script: bundle exec rake beaker
+  - rvm: 2.4.4
```

Comment:
puppet6 should use 2.5.1

Reply:
That version of RVM only affects beaker outside the container environment so no bearing on Puppet Ruby version since Ruby is installed into container using packages.  Still want it to be 2.5.1?  I kept consistent with other beaker tests since only interacts with beaker.

Reply:
Yeah, because we want the testing environment to match the version we actually use. This is useful so any other ruby tooling used around your Puppet setup can be on the same version.